packages/plugingen/src/commands/generate/phases/plugin-actions/index.ts

```diff
@@ -46,49 +46,53 @@ export const PluginActionsGenPhase: Phase = async (input) => {
     .map((n) => {
       const argsParamString =
         n.inputs.length > 0
-          ? dedent`{ 
-                  args, 
-                  overrides, 
-                  account = client.account 
+          ? dedent`{
+                  args,
+                  overrides,
+                  account = client.account
```

Comment:
looks like `context` is missing here

Reply:
[I don't think I need it because it's not used in ](https://github.com/alchemyplatform/aa-sdk/pull/514/files#r1562043130)

Reply:
seems like a bug then because the original code does include it: https://github.com/alchemyplatform/aa-sdk/pull/514/files#diff-67ce60c0c18c8684ea92aadf41e3e203bb0b198f3bb0615b56fe91cb72ce612eR53

Reply:
can you add the `context` parameter to both of these lines? it should match the code in the other `plugin-actions/index` file